image.c

```diff
@@ -293,14 +300,223 @@ bool img_load_gif(img_t *img, const fileinfo_t *file)
 }
 #endif /* HAVE_GIFLIB */
 
+
+#if HAVE_LIBWEBP
+bool img_load_webp(img_t* img, const fileinfo_t* file)
+{
+	bool err = false;
+	FILE *webp_file;
+	size_t file_size;
+	WebPData data;
+
+	Imlib_Image im = NULL;
+	struct WebPAnimDecoderOptions opts;
+	WebPAnimDecoder *dec;
+	struct WebPAnimInfo info;
+	uint8_t *buf = NULL;
+	int ts;
+	const WebPDemuxer *demux;
+	WebPIterator iter;
+	uint32_t flags;
+	uint32_t delay;
+
+	// Check if file exists (and we are able to read it)
+	if (access(file->path, R_OK) == -1) {
+		error(0, 0, "%s: Error opening webp image", file->name);
+		return false;
+	}
```

Comment:
I don't believe this is needed. `check_add_file` already checks for this.

Reply:
will start tests on removing this and checking that nothing breaks, in theory it should not break as this is redundant.

Reply:
doing some tests this is certainly redundant and can be removed without any problem